x-pack/plugins/observability/public/components/shared/exploratory_view/configurations/default_configs.ts

```diff
@@ -22,15 +22,21 @@ export const getDefaultConfigs = ({
   indexPattern,
   reportConfigMap,
 }: Props): SeriesConfig => {
-  let configResult: SeriesConfig;
+  let configResult: SeriesConfig | undefined;
 
-  reportConfigMap[dataType].some((fn) => {
+  reportConfigMap[dataType]?.some((fn) => {
     const config = fn({ indexPattern });
     if (config.reportType === reportType) {
       configResult = config;
     }
     return config.reportType === reportType;
   });
 
-  return configResult!;
+  if (!configResult) {
+    throw new Error(
+      `No report config provided for dataType: ${dataType} and reportType: ${reportType}`
```

Comment:
Do error messages not require translation?

Reply:
AFAIK if they're surfaced to the user, they should be translated.

Reply:
Yeah this isn't a user facing. More of a dev error, i will add the comment to clarify.